fetchcode/github_tags.py

```diff
@@ -0,0 +1,46 @@
+# fetchcode is a free software tool from nexB Inc. and others.
+# Visit https://github.com/nexB/fetchcode for support and download.
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and http://aboutcode.org
+#
+# This software is licensed under the Apache License version 2.0.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: 
+# http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import requests
+from urllib.parse import urlparse
+
+
+def build_url(base_url, path):
+    """
+    Builds URL according to GitHub API 
+    """
+    return base_url + path + '/tags'
+
+
+def get_tags(url):
+    """
+    Takes Github VCS URL as input and provides 
```

Comment:
Same comment as above: use imperative instead.
```
Return a list of git tags  given the `url` to a Github repository
```

Reply:
Okay got this